keyboards/converter/modelm101/modelm101.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 iw0rm3r
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "modelm101.h"
+
+void keyboard_pre_init_kb(void) {
+  /* Setting status LEDs pins to output and +5V (off) */
+  DDRB  |= (1<<4) | (1<<5) | (1<<6);
```

Comment:
Could you update these to use the GPIO commands, instead? 
https://docs.qmk.fm/#/internals_gpio_control

Eg, these should be:
```suggestion
  setPinOutput(B4);
  setPinOutput(B5);
  setPinOutput(B6);
```